packages/cli/test/integration/project.api.test.ts

```diff
@@ -764,18 +843,14 @@ describe('PATCH /projects/:projectId', () => {
 
 			const memberAgent = testServer.authAgentFor(testUser2);
 
-			const resp = await memberAgent.patch(`/projects/${teamProject.id}`).send({
-				name: teamProject.name,
-				relations: [
-					{ userId: testUser1.id, role: 'project:viewer' },
-					{ userId: testUser2.id, role: 'project:admin' },
-					{ userId: testUser3.id, role: 'project:admin' },
-				] as Array<{
-					userId: string;
-					role: ProjectRole;
-				}>,
-			});
-			expect(resp.status).toBe(200);
+			const resp1 = await memberAgent
```

Comment:
If we keep the relations field above, these test cases would still be valid. I guess the question is, do we want to get rid of the old API as well or extend it with a new API. If we want to keep the relations field in the patch of the project as well, we should also keep the tests for it. I'm not sure what is the best approach for it though. 😅 

This goes for this whole test file.